ui/src/config/section/infra/clusters.js

```diff
@@ -131,8 +130,7 @@ export default {
       message: 'label.outofbandmanagement.disable',
       dataView: true,
       show: (record) => {
-        return !(record.resourcedetails && record.resourcedetails.outOfBandManagementEnabled &&
-          record.resourcedetails.outOfBandManagementEnabled === 'false')
+        return !(record.resourcedetails?.outOfBandManagementEnabled === 'false')
```

Comment:
Should be `return !(record?.resourcedetails?.outOfBandManagementEnabled === 'false')`